### PR TITLE
Adding tooltip to categories manager search field

### PR DESCRIPTION
**Pull request by infograf768**
As title says.

After patch, one should get:

screen shot 2016-04-18 at 14 54 23

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_categories.ini [hunk 1]
```diff
@@ -43,6 +43,7 @@ COM_CATEGORIES_FIELD_PARENT_LABEL="Parent"
 COM_CATEGORIES_FIELDSET_DETAILS="Category Details"
 COM_CATEGORIES_FIELDSET_PUBLISHING="Publishing"
 COM_CATEGORIES_FIELDSET_RULES="Permissions"
+COM_CATEGORIES_FILTER_SEARCH_DESC="Search title or alias. Prefix with ID: to search for a category ID."
```
Comment (andrepereiradasilva) [inline administrator/language/en-GB/en-GB.com_categories.ini]: You can search for "id:" or "author:" or free text.
This tooltip should reflect that.

See https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_categories/models/categories.php#L233-L246

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 536f2492e7d3ce98b0dba02b522f8ed7fd54c2ef<br /><br />Tested with success. Before patch, no tool tip for the search field in categories list, after patch, same tooltip as in the articles list.

Suggestion: I would prefer the "ID:" to be quoted in the language string, so it is (for English):

> ... Prefix with "ID:" to ... 

or

> ... Prefix with 'ID:' to ...

whatever we normally use to quiote strings in tool tips.

But thast would be another PR.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9974">issues.joomla.org/joomla-cms/9974</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 536f2492e7d3ce98b0dba02b522f8ed7fd54c2ef<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9974">issues.joomla.org/joomla-cms/9974</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9974">issues.joomla.org/joomla-cms/9974</a>.</sub>

**Comment by infograf768** [comment]
@andrepereiradasilva 
you are right. I suggest to do a specific pr adding author for all places where it is set in the list query (I saw it in articles for example) At the same time I also saw it was missing for contacts

**Comment by andrepereiradasilva** [comment]
it's missing in most views i think.

**Pull request merged**